src/resource_clients/run.js

```diff
@@ -92,13 +92,26 @@ class RunClient extends ResourceClient {
 
     /**
      * https://docs.apify.com/api/v2#/reference/actor-runs/resurrect-run/resurrect-run
+     * @param {object} [options]
+     * @param {object} [options.contentType]
+     * @param {string} [options.build]
+     * @param {number} [options.memory]
+     * @param {number} [options.timeout]
+     * @param {boolean} [options.resendRunCreatedWebhook]
```

Comment:
I don't know if this is a good idea to have a property for one specific webhook event. This could get out of hand pretty fast. But that's a question of the API design, not really the client.

Reply:
I mean, it looks like the property is a hotfix of something. Which it technically is.

Reply:
We have 2 options - we will do a breaking change and start sending the "RUN STARTED" event webhook on each resurrect or we make this opt-in in a non-breaking way

Reply:
Yeah, agree making it opt-in. But can we make it `resendWebhooks: true` or `resendWebhooks: ['RUN.CREATED' , 'RUN.FAILED']? This option is a bit weird.

Reply:
The thing is that all other except created are being resent always, just created one not

Reply:
Yeah I remember, so maybe the question is if that behavior is correct and if we should allow the user to tweak it.

I mean, it's not critical at all, but it's definitely super weird to have a super specific option like that 😄 If you're OK with it in the API, I'm ok with it in the client.